libmscore/textedit.cpp

```diff
@@ -455,6 +456,11 @@ bool TextBase::edit(EditData& ed)
                         }
                   }
             }
+            if (ctrlPressed && altPressed) {
```

Comment:
If you want the shortcut to be configurable, you could add a configurable shortcut in cmd.cpp like you did in the beginning, and compare that shortcut with what's pressed here.
Same for your other PR.

Reply:
ie in shortcut.h there is a Shortcut class, with a `keys` method. you could get the `Shortcut` for this action and compare the key sequences.

Reply:
Agreed, a configurable shortcut would be nice.

Reply:
Any updates @JLWALTENER?

Reply:
@vpereverzev It seems this PR is in Draft mode, the consensus seems to use the SMuFL symbols as preconised by @Jojo-Schmitz and I, for now the F2 palette present the SMuFL ones and the U+035C more availiable than the U+203F.
May be Asmatzaile will update his PR.

Reply:
I updated the PR to use the SMuFL symbol instead of the unicode one.

I think I understand how I should change the shortcut.h and shortcut.cpp files to make the shortcut configurable, but I don't know what I should write in the textedit.cpp file to let it know about those shortcuts and thus execute the action.

Reply:
Thanks, the SymID is mapped to the right unicode codepoint, could it be possible to have a test ?

Reply:
I don't know how to test it. I tried compiling the code but I get an error (can't compile, install a generator) although I installed all the requirements specified on the developer's handbook

Reply:
It doesn't build, there's a missing closing brace